webf/lib/src/devtools/modules/css.dart

```diff
@@ -85,6 +110,11 @@ class InspectCSSModule extends UIInspectorModule {
       // Use styleSheetId to identity element.
       int nodeId = edit['styleSheetId'];
       String text = edit['text'] ?? '';
+      // parse should remove style property
```

Comment:
`// parse should remove style property` --> `// Use parser to remove annotated styles.`